api/index.js

```diff
@@ -95,7 +96,7 @@ function parseMemeBody(body) {
         const text = childrenToText(paragraphs);
 
         if (text && text.trim() !== '') {
-            return { name, about: text };
+            return { name, about: text, url: url, image: image};
```

Comment:
Same as other comment about property names, it would likely also be beneficial to create a Meme class which would hold its data instead of manually dealing with objects each time.